source/documentation/standards/hosting.html.md.erb

```diff
@@ -0,0 +1,78 @@
+---
+title: How to host services
+last_reviewed_on: 2020-09-01
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+At MOJ you must use the [Cloud
+Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) to
+host your service unless you're advised otherwise by a member of [the
+hosting team](https://peoplefinder.service.gov.uk/teams/hosting). Your
+team can get started on the Cloud Platform without letting anyone know;
+just follow [the Cloud Platform documentation
+](https://user-guide.cloud-platform.service.justice.gov.uk/) and get in
+touch with the Cloud Platform team if you need help.
+
+The Cloud Platform is a modern hosting platform for digital services:
+
+- It makes it quick and easy to 'just deploy an app' and you get easy
```

Comment:
Can we split this into two paras to emphasise deploying and operating a service. Maybe something like:

- It makes it quick and easy to 'just deploy an app'
- Provides team with the features to operate their service, such as scalability, zero downtime deploys, security scanning, monitoring, logging, and more, all setup for you with sensible defaults